docs/readthedocs/models/input/additionaldata/timeseries.md

```diff
@@ -32,10 +32,12 @@ The following different values are available:
      - Electrical active and reactive power
      
    * - `HeatAndPValue`
-     - Combination of thermal power (e.g. in kW) <br> and electrical active power (e.g. in kW)
+     - | Combination of thermal power (e.g. in kW) 
+       | and electrical active power (e.g. in kW)
      
    * - `HeatAndSValue`
-     -  Combination of thermal power (e.g. in kW) <br> and electrical active and reactive power (e.g. in kW and kVAr)
+     - | Combination of thermal power (e.g. in kW) 
+       | and electrical active and reactive power (e.g. in kW and kVAr)
```

Comment:
[This does not work...](https://powersystemdatamodel--1130.org.readthedocs.build/en/1130/models/input/additionaldata/timeseries.html)